tutorials/introduction-gee-python-api-guiattard/tutorial.ipynb

```diff
@@ -0,0 +1,1244 @@
+{
```

Comment:
<ul><li>Please remove the output cell</li><li>Please add extra info to print statements to let people know what is being printed and that they may need to wait; suggestion:</li></ul><p> </p><pre class="ql-syntax" spellcheck="false">from&nbsp;IPython.display&nbsp;import&nbsp;Image

#&nbsp;Create&nbsp;the&nbsp;url&nbsp;associated&nbsp;to&nbsp;the&nbsp;Image&nbsp;you&nbsp;want
url&nbsp;=&nbsp;LST_im.getThumbUrl({'min':&nbsp;10,&nbsp;'max':&nbsp;30,&nbsp;'dimensions':&nbsp;512,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'region'&nbsp;:&nbsp;roi,
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;'palette':&nbsp;['blue',&nbsp;'yellow',&nbsp;'orange',&nbsp;'red']})
print('Thumbnail&nbsp;URL:',&nbsp;url)

#&nbsp;Display&nbsp;a&nbsp;thumbnail&nbsp;of&nbsp;Land&nbsp;Surface&nbsp;Temperature&nbsp;in&nbsp;France
print('\nPlease&nbsp;wait&nbsp;while&nbsp;the&nbsp;thumbnail&nbsp;loads,&nbsp;it&nbsp;may&nbsp;take&nbsp;a&nbsp;moment...')
Image(url&nbsp;=&nbsp;url)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/254/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Done</p>